packages/core/src/render3/discovery_utils.ts

```diff
@@ -221,4 +274,83 @@ export function getHostElement<T>(directive: T): Element {
 export function getRenderedText(component: any): string {
   const hostElement = getHostElement(component);
   return hostElement.textContent || '';
-}
\ No newline at end of file
+}
+
+export function loadLContextFromNode(node: Node): LContext {
+  if (!(node instanceof Node)) throw new Error('Expecting instance of DOM Node');
+  return loadLContext(node) !;
+}
+
+export interface Listener {
+  name: string;
+  element: Element;
+  callback: (value: any) => any;
+  useCapture: boolean|null;
+}
+
+/**
+ * Retrieves a list of DOM listeners.
+ *
+ * ```
+ * <my-app>
+ *   #VIEW
+ *     <div (click)="doSomething()">
+ *     </div>
+ * </mp-app>
+ *
+ * expect(getListeners(<child-comp>)).toEqual({
```

Comment:
This should be `<div>`, no? I don't see a `<child-comp>` here.

```suggestion
 * expect(getListeners(<div>)).toEqual({
```

Reply:
fixed